cloudinfo/service.go

```diff
@@ -33,6 +35,7 @@ type CloudInfoService struct {
 	containerClient           containerClient
 	regionsData               []RegionData
 	lock                      sync.Mutex
+	ListDeployablesResponse   *clouddatabasesv5.ListDeployablesResponse
```

Comment:
The CloudInfoService struct is used to store private pointers to the actual services, not their functions. The pointer here should be private (lower case) and point to an interface that exposes the service `*CloudDatabasesV5`.

You should have an interface created for `CloudDatabasesV5` (see other examples below) and put any functions you wish to use in that interface.

The service will then get initialized in the NewCloudInfoSvc functions below.

The interface usage will allow you to properly "mock" the CloudDatabasesV5 for unit tests.